packages/vue/src/components/atoms/SfInput/SfInput.stories.js

```diff
@@ -61,7 +61,7 @@ storiesOf("Atoms|Input", module)
       :error-message="errorMessage"
       :required="required"
       :disabled="disabled"
-      aria-label="Input label"
+      :aria-label="'Input' + label"
```

Comment:
I think it would be better with 'Enter' + label -> this will be more descriptive for the user.

Reply:
@martaradziszewska in this case, I would stay with my solution :) first, assistive technology says what it is and then says what a user can do with it

Reply:
@justyna-13  You don't need to add "input" in the label as screen readers already announce an element's role. Look here: https://www.aditus.io/aria/aria-label/ or e.g on GAP. Anyway, do we need aria-label here? If the element has text or in this case label content, you don't need to add aria-label, as it is redundant. Wdyt ? :)

Reply:
ok, deleted :)